extension/main.js

```diff
@@ -85,18 +85,21 @@
 
 		const ghTab = {url};
 
-		if (window.GitHubNotify.settings.get('count') > 0) {
```

Comment:
Alright. But you still need to check the count in `if (window.GitHubNotify.settings.get('useParticipatingCount')) {`

See: https://github.com/sindresorhus/notifier-for-github-chrome/pull/49